Updated Hybrid/SQL Exception

jamesche75 opened the pull request:
We have fixed the issue in the updated HC Manager, so I edited the answer to point to the HC Manager download.

azurecla commented:
Hi __@magaal__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

squillace commented:
@szarkos to note PR. Thanks!!! We really appreciate the fix.

jamesche75 committed:
Updated Hybrid/SQL Exception 

We have fixed the issue in the updated HC Manager, so I edited the answer to point to the HC Manager download.

GitHubber17 committed:
Update app-service-web-configuration-and-management-faq.md
